index.html

```diff
@@ -42,7 +47,7 @@ <h2>I am a software developer</h2>
             <div class="arrow">
                 <img src="media\Disabled.png" alt="arrow">
             </div>
-        
+        </div>
```

Comment:
- [ ] please kindly follow the design as much as possible.` my recent works` heading should appear like in the original design on the desktop version
yours
![Screenshot (822)_LI](https://user-images.githubusercontent.com/65298048/220936423-4b0db12d-63dd-4960-8d6d-0bac12cab961.jpg)
figma
![Screenshot (821)](https://user-images.githubusercontent.com/65298048/220936468-417048fb-8b49-4977-89ee-219445564e96.png)